README.md

```diff
@@ -171,6 +170,44 @@ var userInfo = new RaygunIdentifierMessage("12345")
 
 Log.ForContext("CustomUserInfoProperty", userInfo, true).Error(new Exception("random error"), "other information");
 ```
+
+### onBeforeSend
+`type: Action<OnBeforeSendParameters>`
+
+`default: null`
+
+The field allows users to manipulate the data to be sent to Raygun.
+This is a function that is executed by the raygun client's `CustomGroupingKey` event, and happens at the end of the `OnCustomGroupingKey` event callback.
```

Comment:
This sentence has information that's likely not relevant/known by most users, I'd suggest removing it.